### ISSUE TITLE
Cannot import zip with Vorto models

### ISSUE THREAD

**Issue by hristobojilov** [eclipse/vorto#1671]
I have a file _models.zip_ file with the following structure:
```
/datatypes/com.mycompany.project2.Time_1.0.0.type
/datatypes/com.mycompany.project1.Time_1.0.1.type
```
and when I try to import it by using standard Vorto model importer I'm getting strange error:
 **Error - Uploaded File does not contain any Vorto files.**
But the zip file obviously does and these files can be imported individually without zip.
Here are my models:
```

vortolang 1.0

namespace com.mycompany.project1
version 1.0.1

entity Time {
}
```
and:

```
vortolang 1.0

namespace com.mycompany.project2
version 1.0.0

entity Time {
}
```

**Comment by hristobojilov** [comment]
What should be the structure of the zip supported by the standard Vorto model importer?

**Comment by aedelmann** [comment]
Any structure should be fine. In your case it should be working.

**Comment by hristobojilov** [comment]
Here are the stack-traces of the error itself:

```
[nio-8443-exec-6] o.e.v.r.web.importer.ImportController    : uploadModel: [models.zip]
java.lang.IllegalStateException: A resource with the normalized URI 'file:/C:/github/vorto/repository/repository-server/Time.type' was already registered. 
The resource with the URI 'org.eclipse.xtext.linking.lazy.LazyLinkingResource@60fbd416 uri='Time.type'' is no longer registered with the normalized form.
...
        at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.createResource(ResourceSetImpl.java:423) ~[org.eclipse.emf.ecore-2.15.0.jar!/:na]
        at org.eclipse.xtext.resource.SynchronizedXtextResourceSet.createResource(SynchronizedXtextResourceSet.java:32) ~[org.eclipse.xtext-2.16.0.jar!/:na]
        at org.eclipse.vorto.repository.core.ModelResource.toDSL(ModelResource.java:91) ~[repository-core-0.11.0-SNAPSHOT.jar!/:na]
        at org.eclipse.vorto.repository.importer.impl.VortoModelImporter.preValidation(VortoModelImporter.java:96) ~[repository-core-0.11.0-SNAPSHOT.jar!/:na]
        at org.eclipse.vorto.repository.importer.AbstractModelImporter.upload(AbstractModelImporter.java:110) ~[repository-core-0.11.0-SNAPSHOT.jar!/:na]
        at org.eclipse.vorto.repository.web.importer.ImportController.uploadModel(ImportController.java:92) ~[repository-core-0.11.0-SNAPSHOT.jar!/:na]
```

**Comment by aedelmann** [comment]
Hi @hristobojilov 
yeah its a bug. Will fix it along the way our release 0.11.
Thx.
